src/electron/main.ts

```diff
@@ -23,15 +23,15 @@ async function createWindow() {
         minWidth: 450,
         minHeight: 100,
         frame: isLinux,
-        titleBarStyle: "hidden",
+        titleBarStyle: "hiddenInset",
         backgroundColor: "#272B30",
         show: false,
         icon: "app-icons/icon.png"
     };
     windowOptions.webPreferences = {
         nodeIntegration: true,
-        webSecurity: false,
-        enableRemoteModule: true
+        enableRemoteModule: true,
+        webSecurity: true
```

Comment:
This seems to prevent opening existing projects on Windows. Ideally we should have webSecurity enabled though, so we'll need to investigate this more

Reply:
Thanks @stew-ro! I wasn’t able to test Windows in much detail yet.

What I found out already is that Windows is not using the configured menu... It shows the default Electron Help menu which is not configured in code. (already a problem in current master possibly)